projects/vdk-plugins/vdk-notebook/README.md

```diff
@@ -1,19 +1,38 @@
-# vdk-notebook
+# Versatile Data Kit Notebook
```

Comment:
I like the change and think it is a good idea
I do not think that this README changes are enough and think we should include more information both here and in the VEP but this can be done in a separate effort.